src/scripts/skipper-docs-name.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
I am not sure what this script does based on its current name alone. Maybe we should add a header comment OR start a README in this `src/scripts` dir with a description for each script (of course just priming that README w/ this script and we can add the others in as we go or later). Wdyt?